COMMUNICATION.md

```diff
@@ -1,58 +1,63 @@
 
 ## When to pay attention to the guidelines
-Obviously we're communicating all the time, we can't read this article every time we're having something come out of our mouths or keyboards. That would result in silencing the world 🤐, and thats the opposite of the intention of these guidelines.
+Obviously we're communicating all the time. We can't read this document every time we're having something come out of our mouths or keyboards. That would result in silencing the world 🤐, and thats the opposite of the intention of these guidelines.
 
-> - **Only in professional communication**
-When making jokes, talking about your favorite tea, or planning a team members birthday party surely a lot of things can go wrong in communication as well. But these guidelines are meant to solve issues in our professional comunication.
-> - **When writing short replies**
-When you write a reply of less than ~5 words, have an alarmbel in the back of your head 🔔  Do you agree on what you are replying to? Then it’s probably fine! “Great job”, “Couldn’t have put it better”,  “toats awesome!” In every other case though: 🚨  It is probably necessary to elaborate your reply according to the guidelines below. It may seem inneficient to be so verbose in our reply, but it will avoid misunderstandings which can result in way greater inneficiency.
+- **Only in professional communication**
+When making jokes, talking about your favourite tea, or planning a team members birthday party surely a lot of things can go wrong in communication as well. But these guidelines are meant to solve issues in our professional communication, like pull request comments, code review and more thorough discussions.
 
+- **When writing short replies**
+When you write a reply of less than ~5 words, have an alarm bell in the back of your head 🔔. Is it just tacit agreement? Then it’s probably fine! “Great job”, “Couldn’t have put it better”,  “toats awesome!” are all fine. In every other case though: 🚨!
 
-## Giving critique or asking a question 
-Start your reply with repeating **what** you are replying on in your own words.
+It good practice to elaborate. It may seem inefficient to be so verbose in our replies, but it will avoid misunderstandings, which can result in way greater inefficiency. Avoid being pennywise, pound foolish.
 
-### So not:
->- Don't use optional here. 
->- What’s your suggestion?
+## Criticising or asking questions
+A good idea is to start by repeating **what** you are replying to in your own words. This avoids cases where due to the high volume of messages, the context of your comment is lost. It also helps you make sure you yourself fully understood the problem, and didn’t misread anything.
```

Comment:
I think being more indirect here comes off more polite. I don't expect anyone to go all _lawyer_ on us and say "Ah! But the guideline just implies it's a good idea, being polite isn't mandatory!" 😄